manim/mobject/number_line.py

```diff
@@ -50,6 +94,53 @@ def __init__(
         x_max=config["frame_x_radius"],
         **kwargs
     ):
+        """
+
+        Parameters
+        ----------
+        color : :class:`~.Colors`, optional
+            the color for the number line, by default LIGHT_GREY
+        unit_size : :class:`int`, optional
```

Comment:
I would love to have typing the python code as well :) Can you?

Reply:
sure; let me do that.